app/lib/database/model/Users.js

```diff
@@ -0,0 +1,14 @@
+import { Model } from '@nozbe/watermelondb';
+import { field } from '@nozbe/watermelondb/decorators';
+
+export default class User extends Model {
```

Comment:
I've created this file as `Users`, since we already have a file called `User` on our models that refers to logged user.
I'm thinking about create a new folder to separate Models of the Default and Servers database.
WDYT?

Reply:
I agree.